### ISSUE TITLE
The app crashes importing with cocoapods and "use_frameworks!"

### ISSUE THREAD

**Issue by serluca** [PTEz/LumberjackConsole#10]
# Podfile example

```
platform :ios, '7.0'
use_frameworks!
pod 'LumberjackConsole', :configurations => ['Debug']
```
# Crash report

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException',
 reason: 'Could not find a storyboard named 'LumberjackConsole' in bundle NSBundle ...
```

**Comment by AmauryVela** [comment]
Its happening to me Could not find a storyboard named 'LumberjackConsole' in bundle NSBundle

**Comment by serluca** [comment]
This happens because the [podspec](https://github.com/PTEz/LumberjackConsole/blob/master/LumberjackConsole.podspec) still points to the version `2.4.3`.
@rivera-ernesto can you please push a new version?

In the mean time you can use 
`pod 'LumberjackConsole', :git => 'git@github.com:serluca/LumberjackConsole.git'` (tested)
 or
 `pod 'LumberjackConsole', :git => 'git@github.com:PTEz/LumberjackConsole.git', :branch => 'master'` (didn't test) to fix this problem

**Comment by rivera-ernesto** [comment]
Sure, I will take a look as soon as I can. Thanks for the suggested fixes too.

**Comment by johndpope** [comment]
Hi Eneresto,

How are you?  I'm learning to be patient these days.
Is there something I can help with? 

JP

**Comment by rivera-ernesto** [comment]
I haven't used `use_frameworks!` with there libraries yet.

It would help to add that to the demo app's Podfile and try to fix the errors.

**Comment by johndpope** [comment]
It pains me to say this - but I'm afraid without the use_frameworks! compatibility - I can't recommend the team use this codebase. Found this (crude) alternative - https://github.com/burczyk/UIForLumberjack

**Comment by rivera-ernesto** [comment]
Fixed since 3.0.0.

**Comment by johndpope** [comment]
thank you.
